bootstrap.sh

```
@@ -1 +1,20 @@
 #!/bin/bash
+
```

Make the script more robust: start with `set -euo pipefail` (or `set -e`) to fail fast on errors; check prerequisites (`command -v kind >/dev/null` and `command -v kubectl >/dev/null`) and print helpful messages. Consider adding explicit `kubectl apply -n <ns>` invocations so resources are applied to the intended namespaces.